src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("file-work-items", HelpText = "Send SARIF results to a work item tracking system such as GitHub or Azure DevOps")]
+    public class FileWorkItemsOptions
+    {
+        [Option(
+            't',
+            "target-type",
+            HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
+            Required = true)]
+        public string TargetType { get; internal set; }
```

Comment:
>TargetType [](start = 22, length = 10)

Mike's comment during the call made me think - since this is exposed to the user, yes this could be interpreted as work item type (e.g. bug/task/issue). Here are some other suggestions:

- filing-target
- version-control-type
- bug-manager
- issue-reporter

Reply:
I think you're on the right track! We already use "filing target" to mean "the place the bugs are going" but how about `filing-target-type`?

---
In reply to: [306055034](https://github.com/microsoft/sarif-sdk/pull/1580#discussion_r306055034) [](ancestors = 306055034)

Reply:
you have my vote on filing-target-type :)

---
In reply to: [306060353](https://github.com/microsoft/sarif-sdk/pull/1580#discussion_r306060353) [](ancestors = 306060353,306055034)